master/internal/rm/kubernetesrm/pods.go

```diff
@@ -327,37 +331,26 @@ func readClientConfig(credsDir string) (*rest.Config, error) {
 		return rest.InClusterConfig()
 	}
 
-	// A special case for rapid determined+k8s development: build a rest.Config from a specially
-	// packed directory with the same information.  Our tools/scripts/fetch-k8s-creds.sh script can
-	// create such a directory, with server, token, and ca.crt files.
+	if parts := strings.Split(kubeconfigPath, string(os.PathSeparator)); parts[0] == "~" {
+		logrus.Infof("expanding kubeconfig path from %s to %s", kubeconfigPath, kubeconfigPath)
+		parts[0] = homedir.HomeDir()
+		kubeconfigPath = filepath.Join(parts...)
```

Comment:
Do we want to maybe add the `logrus` comment down here and save the old `kubeconfigPath`? Right now, the log prints the same path twice

Reply:
yep good call. i had it there before, oops.

Reply:
refactor gone awry

Reply:
fixed